src/visualization/components/View.test.tsx

```diff
@@ -67,6 +67,19 @@ jest.doMock('src/visualization/contextLoader', () => {
   }
 })
 
+jest.mock('src/flows/index', () => {
+  return {
+    PROJECT_NAME: 'NoteBook',
+    DEFAULT_PROJECT_NAME: 'Untitled Notebook',
+  }
+})
+
+jest.mock('src/flows/templates/index', () => {
+  return {
+    TEMPLATES: {},
+  }
+})
+
```

Comment:
The mocks here are because of the changes to `useStaticLegend` which pulls in the `PipeContext` from Notebooks. This cascades into other dependencies being pulled in, eventually leading to a `require.context` which Jest cannot handle.

By mocking out the two dependencies that call `require.context`, this test file runs correctly again, just like before.